src/azul/files.py

```diff
@@ -8,9 +8,45 @@
 import os.path
 import tempfile
 from typing import (
+    Tuple,
     Union,
 )
 
+composite_extensions = [
+    'bam.bai',
+    'csv.gz',
+    'csv.zip',
+    'fastq.gz',
+    'mtx.gz',
+    'mtx.zip',
```

Comment:
See above. Have you seen this in the wild?

Reply:
```
(azul.sc) daniel@Crispin ~/azul.sc $ grep 'mtx.zip' files-* | head
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/005d611a-14d5-4fbf-846e-571a1f874f70.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/091cf39b-01bc-42e5-9437-f419a66c8a45.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/116965f3-f094-4769-9d28-ae675c1b569c.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/2043c65a-1cf8-4828-a656-9e247d4e64f1.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/4a95101c-9ffc-4f30-a809-f04518a23803.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/4d6f6c96-2a83-43d8-8fe1-0f53bffd4674.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/8185730f-4113-40d3-9cc3-929271784c2b.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/abe1a013-af7a-45ed-8c26-f3793c24a1f4.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/c4077b3c-5c98-4d26-a614-246d12c2e5d7.homo_sapiens.mtx.zip
files-cgm_dcp1.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/cgm_dcp1/data/cc95ff89-2e68-4a08-a234-480eca21ce79.homo_sapiens.mtx.zip
(azul.sc) daniel@Crispin ~/azul.sc $ grep 'mtx.zip' files-* | wc -l
      14
```